.github/workflows/main.yml

```diff
@@ -155,3 +155,10 @@ jobs:
           CHROMIUM: true
         run: |
           npm run unit
+
+      - name: Run unit tests on Firefox
+        env:
+          FIREFOX: true
+          MOZ_WEBRENDER: 0
+        run: |
+          npm run funit
```

Comment:
Lets see if that works. Last time the check on Windows failed because the environment variable for `PUPPETEER_PRODUCT` could not be set.

Reply:
I think that I found the problem and pushed a follow-up commit for it.